WcaOnRails/app/assets/stylesheets/teams.scss

```diff
@@ -0,0 +1,20 @@
+// Place all the styles related to the teams controller here.
+// They will automatically be included in application.css.
```

Comment:
Lol, auto-generated file.

On Mon, Mar 21, 2016 at 4:41 AM, Jeremy Fleischman <notifications@github.com
> wrote:

> In WcaOnRails/app/assets/stylesheets/teams.scss
> <https://github.com/cubing/worldcubeassociation.org/pull/390#discussion_r56788430>
> :
>
> > @@ -0,0 +1,20 @@
> > +// Place all the styles related to the teams controller here.
> > +// They will automatically be included in application.css.
>
> This comment is wrong, lol. Please remove all 3 lines as we do everywhere
> else.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/390/files/4cfc692de289d2d4d24e03be7e8db4b02da2a89d#r56788430>
>